content/building-a-simple-keras-and-deep-learning-rest-api.md

```diff
@@ -0,0 +1,390 @@
+Title: Building a simple Keras + deep learning REST API
+Date: 2018-01-29
+Category: Tutorials
+Author: Adrian Rosebrock
+
+_This is a guest post by Adrian Rosebrock. Adrian is the author of [PyImageSearch.com](https://www.pyimagesearch.com/), a blog about computer vision and deep learning. Adrian recently finished authoring_ [Deep Learning for Computer Vision with Python](https://www.pyimagesearch.com/deep-learning-computer-vision-python-book/)_, a new book on deep learning for computer vision and image recognition using Keras._
+
+In this tutorial, we will present a simple method to take a Keras model and deploy it as a REST API.
+
+The examples covered in this post will serve as a template/starting point for building your own deep learning APIs &mdash; you will be able to extend the code and customize it based on how scalable and robust your API endpoint needs to be.
```

Comment:
This reminds me that I should script the automatic conversion of ` -- ` to `&mdash;` in the markdown renderer.